code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -321,4 +321,29 @@
 	volume = 20
 	amount_per_transfer_from_this = 20
 
-	reagents_to_add = list(/decl/reagent/sanasomnum = 20)
\ No newline at end of file
+	reagents_to_add = list(/decl/reagent/sanasomnum = 20)
+
+/obj/item/reagent_containers/hypospray/autoinjector/bicaridine
+	name = "bicaridine autoinjector"
+	desc = "An autoinjector loaded with bicaridine, a chemical used to treat physical trauma."
+	volume = 15
+	amount_per_transfer_from_this = 15
+
+	reagents_to_add = list(/decl/reagent/bicaridine = 15)
+
```

Comment:
```suggestion
```

Reply:
yea my git's a little bit fucked right now after I had to wipe my pc so ima just try this commit suggestion button